hack/test/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gcr.io/iguazio/golang:1.14
+
+ENV GOPATH=/go
+WORKDIR /v3io-go
+
+
+RUN apt-get update --allow-releaseinfo-change \
+    && apt-get install -y capnproto \
+    && apt-get clean
+
+# copy `go.mod` for definitions and `go.sum` to invalidate the next layer
+# in case of a change in the dependencies
+COPY go.mod go.sum ./
```

Comment:
just making sure my understanding - this command has no effect as we copy everything anyway on the next command..

Reply:
yes, it's here for cache busting